examples/screen_util_example/lib/example.dart

```diff
@@ -12,9 +12,7 @@ Widget exampleBuilder(BuildContext context) {
 
 /// An example [Widget] that uses [ScreenUtil].
 class ScreenUtilExample extends StatelessWidget {
-  const ScreenUtilExample({
-    super.key,
-  });
+  const ScreenUtilExample({super.key});
```

Comment:
I don't think there's a way to enforce this, since the trailing commas rule did not already.
Most of these widgets aren't intended to have other parameters, so the compact syntax is not doing that much harm for now.